src/status_im2/contexts/wallet/edit_account/view.cljs

```diff
@@ -83,16 +86,20 @@
            :card?           true
            :title           (i18n/label :t/address)
            :custom-subtitle (fn [] [quo/address-text
-                                    {:networks [{:network-name :ethereum :short-name "eth"}
-                                                {:network-name :optimism :short-name "opt"}
-                                                {:network-name :arbitrum :short-name "arb1"}]
+                                    {:networks network-details
                                      :address  address
                                      :format   :long}])
            :on-press        (fn []
                               (rf/dispatch [:show-bottom-sheet
-                                            {:content (fn [] [network-preferences/view
-                                                              {:on-save #(js/alert
-                                                                          "calling on save")}])}]))
+                                            {:content (fn []
+                                                        [network-preferences/view
+                                                         {:on-save (fn [chain-ids]
+                                                                     (rf/dispatch [:hide-bottom-sheet])
+                                                                     (save-account
+                                                                      {:account account
+                                                                       :type    :prod-preferred-chain-ids
```

Comment:
Should be condition? (prod/test)

Reply:
@J-Son89

Reply:
it should be a condition but it's okay to leave for now, I will include this in an issue for adding testnet details 👍